src/main/java/com/programmers/domain/Book.java

```diff
@@ -0,0 +1,102 @@
+package com.programmers.domain;
+
+import java.util.List;
+
+public class Book {
+    @Override
+    public String toString() {
+        return "Book{" +
+                "bookId=" + bookId +
+                ", title='" + title + '\'' +
+                ", author='" + author + '\'' +
+                ", totalPageNumber=" + totalPageNumber +
+                ", bookStatus=" + bookStatus +
+                '}';
+    }
+
+    private Long bookId;
+    private String title;
+    private String author;
+    private int totalPageNumber;
+    private BookStatus bookStatus;
+
+    public Book(String title, String author, int totalPageNumber) {
+        this.title = title;
+        this.author = author;
+        this.totalPageNumber = totalPageNumber;
+        this.bookStatus = BookStatus.AVAILABLE;
```

Comment:
```suggestion
        this(null, title, author, totalPageNumber, BookStatus.AVAILABLE);
```

this()를 활용하여 생성자를 통해 초기화용 생성자와 오버로딩 생성자를 구분해서 사용하면 초기화 시점에 대해서 명확하게 인지할 수 있습니다.

Reply:
[[refactor] 5. 생성자 작성 방식 변경](https://github.com/prgrms-be-devcourse/java-library-management/commit/c8fe762d81fcaffc24d0fc68cc437afa36649891)

반영 완료했습니다!